lib/utils/open-url.js

```diff
@@ -39,10 +42,29 @@ const openUrl = async (npm, url, title, isFile) => {
     assertValidUrl(url)
   }
 
+  const isWSL = async () => {
+    try {
+      const { stdout } = await exec('uname -r')
```

Comment:
This is not gonna work, we don't call bare `exec` to spawn things from npm.  This also seems like a pretty heavy-handed way to detect this.

Reply:
This kind of logic should live in `@npmcli/promise-spawn` in any case, that's its whole reason for existing.

Reply:
It looks like we're already detecting wsl in `promise-spawn` https://github.com/npm/promise-spawn/blob/7928731de498e4dcdfab60ecb1d49040fec521a7/lib/index.js#L133-L137